redis.go

```diff
@@ -167,6 +168,7 @@ type Options struct {
 	WriteTimeout time.Duration
 
 	PoolSize    int
+	PoolTimeout time.Duration
```

Comment:
I know that I am bad at documenting my code, but could you please add brief comment explaining this option? Something like `maximum duration before timing out waiting for connection from the pool`.